### PR TITLE
Add find coronavirus support redirect

### PR DESCRIPTION
**Pull request by MuriloDalRi**
[Trello](https://trello.com/c/t4SszItJ/2159-5-set-up-transition-for-find-coronavirus-supportservicegovuk)

### CODE REVIEW

**Comment by thomasleese** [comment]
I don't know if we want this to be a global redirect. @theseanything @seb-szy Will we need to redirect individual routes to the new Smart Answer?

File: data/transition-sites/gds_find_coronavirus_support.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+  site: gds_find_coronavirus_support
+  whitehall_slug: government-digital-service
+  homepage: https://www.gov.uk/find-coronavirus-support
+  tna_timestamp: 20201010101010 # Stub timestamp - site is not in TNA
+  host: find-coronavirus-support.service.gov.uk
+  global: =301 https://www.gov.uk/find-coronavirus-support
+  global_redirect_append_path: true
+
```
Comment (reggieb) [inline data/transition-sites/gds_find_coronavirus_support.yml:9]: The "No new line at the end of file" warning suggests there is white space on line 9 that I think should be removed.

**Comment by theseanything** [comment]
Yup - a global redirect makes sense. We just want to redirect all traffic to the start page for gov.uk/find-coronavirus-support

File: data/transition-sites/gds_find_coronavirus_support.yml (hunk 1 above)
Comment (MuriloDalRi) [reply data/transition-sites/gds_find_coronavirus_support.yml]: @reggieb Thanks, should be fixed now! 👍

**Review by reggieb — APPROVED** [review]
LGTM

**Pull request merged**